gym/envs/toy_text/blackjack.py

```diff
@@ -240,22 +252,15 @@ def get_font(path, size):
         )
         dealer_text_rect = self.screen.blit(dealer_text, (spacing, spacing))
 
-        suits = ["C", "D", "H", "S"]
-        dealer_card_suit = self.np_random.choice(suits)
-
-        if dealer_card_value == 1:
-            dealer_card_value_str = "A"
-        elif dealer_card_value == 10:
-            dealer_card_value_str = self.np_random.choice(["J", "Q", "K"])
-        else:
-            dealer_card_value_str = str(dealer_card_value)
-
         def scale_card_img(card_img):
             return pygame.transform.scale(card_img, (card_img_width, card_img_height))
 
         dealer_card_img = scale_card_img(
             get_image(
-                os.path.join("img", dealer_card_suit + dealer_card_value_str + ".png")
+                os.path.join(
+                    "img",
```

Comment:
Can this be done with a smart string format

Reply:
`os.path.join` is generally a better option than string interpolation methods when it comes to manipulating paths. In principle we only support Unix, but this would have a high chance of breaking Windows support.

A more elegant way would be using pathlib, but it might be overkill here

Reply:
Agreed, I more meant `"{self.dealer_top_card_suit}{self.dealer_top_card_value_str }.png"`. You are right on `os.path.join`

Reply:
fixed